### PR TITLE
Development

### PR DESCRIPTION
**Pull request by kyracq**
- Increase scroll to top button size
- Simplify color change buttons code
https://kyraa-step-2020.appspot.com/

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -290,14 +270,14 @@ p {
   bottom: 20px;
   color: var(--bg-primary);
   cursor: pointer;
-  font-size: 26px;
-  height: 50px;
+  font-size: 39px;
```
Comment (shulijones) [inline portfolio/src/main/webapp/style.css:273]: The button is a lot more noticeable to me now.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -36,31 +36,31 @@
       <div class="color-buttons">
         <li class="nav-item" id="paint">
           <a class="nav-link white">
-            <i class="material-icons">format_paint</i>
+            <i class="material-icons" style="color: var(--text-primary)"> format_paint</i>
```
Comment (shulijones) [inline portfolio/src/main/webapp/index.html:39]: I really like how you simplified this code! It's not just that there's less code, but also, it's so tidy that the actual color of the button itself controls the color of everything else - it prevents any possible mistakes like mixing up which button is assigned to which color.

**Review by shulijones — APPROVED** [review]
LGTM :)

**Review by anicr7 — APPROVED** [review]

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -36,31 +36,31 @@
...
             <span class="link-text">Change color</span>
           </a>
         </li>
         <li class="nav-item color-button">
-          <a class="nav-link coral" onClick="changeAccentColor(this)">
-            <i class="material-icons">stop_circle</i>
+          <a class="nav-link" onClick="changeAccentColor(this)">
+            <i class="material-icons" style="color: var(--coral)">stop_circle</i>
```
Comment (nsri-google) [inline portfolio/src/main/webapp/index.html:45]: In general it is recommended use html for structure and css files for presentation. https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

What you had before was ok.

**Pull request merged**